tests/Doctrine/Tests/Common/Annotations/DocParserTest.php

```diff
@@ -1005,6 +1005,20 @@ public function testSupportClassConstants(string $docblock, $expected): void
         self::assertEquals($expected, $annotation->value);
     }
 
+    public function testSupportSelfAccessorForClassConstants(): void
```

Comment:
this test is not actually testing your patch. It copies your (current) patch in a test. I could revert the AnnotationReader change and still have a green test.

As this feature is implemented at the AnnotationReader level, tis must be tested at that level too.

Reply:
You're right, I added some test.

Reply:
please also remove this test as it does not actually test the feature but a copy of it

Reply:
Done, thank you for your time.